examples/example_perturbation_expression_prediction.py

```diff
@@ -122,19 +168,32 @@
         print("Task inputs loaded from saved files")
     else:
         print("Creating task input directly from dataset...")
-        # Create task input directly from dataset
+        # Create task input directly from dataset with separate fields
         task_input = PerturbationExpressionPredictionTaskInput(
+            adata=dataset.control_matched_adata,
+            target_conditions_dict=dataset.target_conditions_dict,
             de_results=dataset.de_results,
-            var_index=dataset.control_matched_adata.var.index,
-            masked_adata_obs=dataset.control_matched_adata.obs,
-            target_conditions_to_save=dataset.target_conditions_to_save,
-            row_index=dataset.adata.obs.index,
         )
 
-    # Generate random model output
-    model_output: CellRepresentation = np.random.rand(
-        dataset.adata.shape[0], dataset.adata.shape[1]
-    )
+    # Load model data or generate random data
+    if args.model_data_file:
```

Comment:
I don't understand the workflow -- if model data can be loaded, then why would we run the dataset class and load_dataset?

Reply:
Loading the model data is just for testing purposes, thats fine. If so, please add a comment that it will be removed.